<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer / Ocean Coding School</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="90,5,0,5">
       <item>
        <widget class="QVideoWidget" name="view" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="bar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="state">
           <property name="text">
            <string>Media Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="playtime">
           <property name="text">
            <string>Running time</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_stop">
           <property name="text">
            <string>■</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play">
           <property name="text">
            <string>▶</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_forward">
           <property name="text">
            <string>▶▶</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_pause">
           <property name="text">
            <string>❚❚</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_prev">
           <property name="text">
            <string>◀◀</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="vol">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Play List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_del">
         <property name="text">
          <string>Del</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
